app/views/shared/_header.html.erb

```diff
@@ -51,10 +51,10 @@
         </li>
       <% end %>
 
-      <% if current_user.try(:is_admin?) %>
-        <li class="dropdown <%= 'active' if request.path.starts_with?('/admin') %>">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown">Admin <b class="caret"></b></a>
-          <ul class="dropdown-menu">
+      <li class="dropdown <%= 'active' if request.path.starts_with?('/admin') %>">
+        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Admin <b class="caret"></b></a>
```

Comment:
normal users should not see the admin dropdown ... they might get scared and think it's a bug :(